ansible/group_vars/all

```diff
@@ -25,6 +25,9 @@ controller:
   port: 10001
   blackboxFraction: 0.10
 
+invokerHealth:
+  testAction: whisk.system/utils/date
```

Comment:
do we need this test action? the reason is that the catalog isn't usually installed by default (ie wasn't a requirement) but this suggests it must be installed.

the invoker can just have a builtin action string (ping/date) that it can run - do we need this to be an actual action?

there are two things you're potentially testing by making the health action a real action:
1. the invoker has a healthy connection to cloudant
2. the invoker can execute an action

i think these are being conflated and can be tested separately.
did you consider an independent cloudant ping and a precanned action string to invoke?